projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/README.md

```diff
@@ -98,14 +96,25 @@ Note: These commands should be ran from `/jetpack-mu-wpcom` root directory.
 * `pnpm build-production-js` - Build Verbum production code.
 * `pnpm lint` - Check for lint issues in the code.
 * `pnpm run watch` - Watch file changes
-* `jetpack rsync mu-wpcom-plugin` - Sync local files to development environment. This command tool will ask you for the remote destination after your input it in the command line. Ensure the remote path is correct depending on the environment you're targetting. If you're targetting your sandbox, the remote destination should look like this: `USERNAME@HOSTNAME:~/public_html/wp-content/mu-plugins/jetpack-plugin/production`. You will also need to add `define( 'JETPACK_AUTOLOAD_DEV', true );` to mu-plugins/0-sandbox.php. More details for Simple site testing: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#simple-testing]. If you're targetting your WoA site, the remote destination should look like this: `mywoadevsite.wordpress.com@sftp.wp.com:htdocs/wp-content/plugins/jetpack-mu-wpcom-plugin-dev`. More details for WoA testing: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#woa].
+* `jetpack rsync mu-wpcom-plugin` - Sync local files to development environment. This command tool will ask you for the remote destination after your input it in the command line. Ensure the remote path is correct depending on the environment you're targetting. If you're targetting your sandbox, the remote destination should look like this: `USERNAME@HOSTNAME:~/public_html/wp-content/mu-plugins/jetpack-plugin/production`. You will also need to add `define( 'JETPACK_AUTOLOAD_DEV', true );` to mu-plugins/0-sandbox.php. More details for Simple site testing: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#simple-testing]. If you're targetting your WoA site, the remote destination should look like this: `mywoadevsite.wordpress.com@sftp.wp.com:htdocs/wp-content/plugins/jetpack-mu-wpcom-plugin-dev`. More details for WoA testing: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#woa]. 
+
+In most cases you will only need to sync the code to your sandbox, since Verbum is loaded through and on a simple site in all scenarios. There may be a case where you want to confirm that your PHP changes are not negatively affecting WoA sites (although Verbum will only load on Simple Sites, it doesn't hurt to be safe with PHP specifc code ). In this case, you will want to sync the code directly to your WoA using the steps above. 
```

Comment:
`In most cases you will only need to sync the code to your sandbox, since Verbum is loaded through and on a simple site in all scenarios. ` maybe is missing something? 🤔